File: store/dotnetcore.localfile/Models/Answer.cs
namespace dotnetcore.localfile.Models
{
    public class Answer
    {
        public long Id { get; set; }
        public long QuestionId { get; set; }
        public long UpVote { get; set; }
        public string BountyAward { get; set; }
        public string PostContent { get; set; }
        public string AnswerTime { get; set; }
        public string UserLink { get; set; }
        public string UserAvatar { get; set; }
    }
}

File: store/java.mongodbstore/mongodb-store/src/main/java/net/alanwei/robot/models/Answer.java
package net.alanwei.robot.models;


import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class Answer {
    private Long id;
    private Long questionId;
    private Long upVote;
    private String bountyAward;
    private String postContent;
    private String answerTime;
    private String userLink;
    private String userAvatar;
}

